mlem/utils/module.py

```diff
@@ -341,25 +341,29 @@ def get_local_module_reqs(mod) -> List[ModuleType]:
     except (OSError, TypeError):
         logger.debug("Failed to get source of %s", str(mod))
         return []
-    imports: List[Tuple[str, Optional[str]]] = []
+    imports: List[ModuleType] = []
     for statement in tree.body:
         if isinstance(statement, ast.Import):
-            imports += [(n.name, None) for n in statement.names]
+            imports += [
+                importlib.import_module(n.name, None) for n in statement.names
+            ]
         elif isinstance(statement, ast.ImportFrom):
             if statement.level == 0:
                 imp = (statement.module or "", None)
             else:
-                imp = ("." + (statement.module or ""), mod.__package__)
-            imports.append(imp)
+                imp = (
+                    "." * statement.level + (statement.module or ""),
+                    mod.__package__,
+                )
+            imports.append(importlib.import_module(*imp))
 
-    result = [importing.import_module(i, p) for i, p in imports]
```

Comment:
A few points:
- `import_module ` is from importlib which was incorrectly imported
- The proposed fix adds `"." * statement.level` to handle relative imports
- I have changed it to import modules immediately, instead of creating the list once and importing in another iteration

Reply:
Hi @aminalaee, looks right. The only thing - let's keep `from mlem.utils import importing` - this was done to keep all the import utils in a single place to be able to update them later (e.g. we can define custom `importing.import_module` func which won't be simply `importlib.import_module`). Otherwise LGTM!

Also sorry for the delay - had a conference and a flight last week, that got me miss this review. Thanks again for the contribution! 🚀

Reply:
Hey @aguschin I have reverted the import change (although I think it's not adding any benefits, your call)

No worries, and thanks for the review :)
If you have any idea where the submodule with commit can be fetched, I'll be happy to submit a PR.